Owncloud iOs Client/Tabs/FileTab/SelectFolder/SelectFolderViewController.m

```diff
@@ -65,6 +65,7 @@ @implementation SelectFolderViewController
 @synthesize folderView=_folderView;
 @synthesize toolBarLabelTxt = _toolBarLabelTxt;
 @synthesize alert = _alert;
+@synthesize activeUser = _activeUser;
```

Comment:
It is no longer needed to synthesize the property manually, the compiler will do it